service/src/integrationTest/kotlin/fi/espoo/evaka/pis/controller/ParentshipControllerIntegrationTest.kt

```diff
@@ -90,10 +89,10 @@ class ParentshipControllerIntegrationTest : FullApplicationTest(resetDbBeforeEac
         val endDate = startDate.plusDays(200)
         val reqBody =
             ParentshipController.ParentshipRequest(parent.id, child.id, startDate, endDate)
-        controller.createParentship(Database(jdbi), user, clock, reqBody)
+        controller.createParentship(dbInstance(), user, clock, reqBody)
```

Comment:
Should `controller` be renamed as `parentshipController`? That has been the naming convention in all other test classes.

Reply:
I think `controller` is fine in this case, because the test is literally a test for that specific controller (it's even in the class name)